src/app/_services/auth.service.ts

```diff
@@ -0,0 +1,31 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+import { Observable } from 'rxjs';
+
+const AUTH_API = 'http://localhost:8080/api/auth/';
```

Comment:
Use https://ani-be-stg.herokuapp.com/api/ to interact with live API. Refer to [docs](https://ani-be-stg.herokuapp.com/api/docs) for more information.

You might want to use a proxy configuration that automatically routes `/api/` to  https://ani-be-stg.herokuapp.com/api/. See [here](https://stackoverflow.com/questions/37172928/angular-cli-server-how-to-proxy-api-requests-to-another-server). This is done so you do not have to maintain an `AUTH_API` constant.